### PR TITLE
Bump stylelint

### PR DESCRIPTION
**Pull request by ivarnakken**

### CODE REVIEW

File: app/components/Button/Button.css [hunk 1]
```diff
@@ -30,7 +30,7 @@
   }
 
   &:hover:not(:disabled) {
-    box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
+    box-shadow: 0 1px 2px rgba(0, 0, 0, 10%);
```
Comment (LudvigHz) [inline app/components/Button/Button.css:33]: I hate this rule change, but oh well :shrug:

File: app/routes/quotes/components/Quotes.css [hunk 2]
```diff
@@ -123,15 +123,6 @@
   }
 }
 
-.rightQuote {
```
Comment (LudvigHz) [inline app/routes/quotes/components/Quotes.css:126]: Are these all unused?

**Review by LudvigHz — APPROVED** [review]
niiiiice :100:

**Review by SmithPeder — APPROVED** [review]
Looks good, but should still some testing 😂

File: app/components/Button/Button.css (hunk 1 above)
Comment (ivarnakken) [reply app/components/Button/Button.css:33]: I kinda regret not changing the rule and sticking with the decimal format 🤔 I might change back, but not tonight ...

File: app/routes/quotes/components/Quotes.css (hunk 2 above)
Comment (ivarnakken) [reply app/routes/quotes/components/Quotes.css:126]: Indeed

**Pull request merged**